oauth2client/contrib/gce.py

```diff
@@ -134,8 +133,8 @@ def _refresh(self, http_request):
             self._retrieve_info(http_request)
             self.access_token, self.token_expiry = _metadata.get_token(
                 http_request, service_account=self.service_account_email)
-        except httplib2.HttpLib2Error as e:
-            raise client.HttpAccessTokenRefreshError(str(e))
+        except transport.transport_error() as err:
```

Comment:
Yeah, I mean, the transport needs to have some way of communicating connection errors. Are you saying `metadata.get()` should return something like `None` instead of raising an exception?